PCbuild/pcbuild.proj

```diff
@@ -34,7 +34,7 @@
     </Projects2>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <!-- pythonXY.dll -->
+    <!-- pythonX_Y.dll -->
```

Comment:
I thought this was going to be `python3.X.dll` according to the doc changes in the rest of this PR?